yb-voyager/src/query/queryissue/issues_perf.go

```diff
@@ -26,12 +26,12 @@ import (
 )
 
 const (
-	COLUMN_NAME                  = "Column Name"
-	EXISTING_INDEX_SQL_STATEMENT = "Existing Index SQL Statement"
-	CARDINALITY                  = "Cardinality"
-	FREQUENCY_OF_NULLS           = "Frequency of Nulls"
-	VALUE                        = "Value"
-	FREQUENCY_OF_VALUE           = "Frequency of the value"
+	COLUMN_NAME                  = "column_name"
```

Comment:
same as above. Just want to confirm if thats a conscious decision.

Do we follow snakecase throughout for these kind of things(keys in json)?

Reply:
There is a difference between these keys and JSON keys. JSON keys are struct keys, which are mostly camel-cased in our report. In Callhome, they are mostly lowercase. For these map keys, we don't have any standard defined ye, so I just used the snake casing as we already function to convert it to camel casing.